rwlib/CMakeLists.txt

```diff
@@ -41,10 +41,9 @@ SET(RWLIB_SOURCES
 	"source/job/WorkContext.cpp"
 	)
 
-add_library(rwlib
+set_source_files_properties(source/gl/gl_core_3_3.c PROPERTIES COMPILE_FLAGS -w)
```

Comment:
The cpp formats introduce a namespace. Some intermediate header that sorts out the linkage or imports the namespace seems like a reasonable solution.